dojo/tools/appspider/parser.py

```diff
@@ -1,23 +1,21 @@
-
-
-from datetime import datetime
 from xml.dom import NamespaceErr
-
 from defusedxml import ElementTree
-
 from dojo.models import Endpoint, Finding
 import html2text
 import urllib.parse
-
-__author__ = "Jay Paz"
+import logging
 
 
 class AppSpiderXMLParser(object):
     def __init__(self, filename, test):
+        self.items = []
+
+        if filename is None:
+            return
 
         if "VulnerabilitiesSummary.xml" not in str(filename):
-            raise NamespaceErr('Please ensure that you are uploading AppSpider\'s VulnerabilitiesSummary.xml file.'
-                               'At this time it is the only file that is consumable by DefectDojo.')
+            logging.warning('Please ensure that you are uploading AppSpider\'s VulnerabilitiesSummary.xml file.'
```

Comment:
are you sure this is intended behaviour? the warning will not be seen by the end user and the import will continue anyway instead of raising an exception.

Reply:
this check is useless. it force the code to have a real file with a fixed name. it is not what we want.
we want to load data evene without a real file on the disk

Reply:
ah, now I see. I thought it was checking for some XML node to be present. But checking the filename makes no sense, so the whole ìf` block can be removed I think.

Reply:
Don't ask me twice to remove useless code 😄 !
Made the change requested.